internal/server/firm_deputy_hub.go

```diff
@@ -3,29 +3,45 @@ package server
 import (
 	"github.com/ministryofjustice/opg-sirius-supervision-firm-deputy-hub/internal/sirius"
 	"net/http"
+	"strconv"
+	"strings"
 )
 
 type FirmDeputyHubInformation interface {
+	GetFirmDetails(sirius.Context, int) (sirius.FirmDetails, error)
 }
 
 type firmDeputyHubVars struct {
 	Path      string
 	XSRFToken string
 	Error     string
 	Errors    sirius.ValidationErrors
+	FirmDetails sirius.FirmDetails
 }
 
-func renderTemplateForFirmDeputyHub(client FirmDeputyHubInformation, tmpl Template) Handler {
+func renderTemplateForFirmHub(client FirmDeputyHubInformation, tmpl Template) Handler {
```

Comment:
If we are renaming the function to "FirmHub" instead of "FirmDeputyHub" (makes sense), we should do it for the parameter type as well and any mocks.